Tasks/UseNodeV1/task.json

```diff
@@ -0,0 +1,54 @@
+{
+    "id": "31C75BBB-BCDF-4706-8D7C-4DA6A1959BC2",
+    "name": "UseNode",
+    "friendlyName": "Use Node.js Eco-System",
+    "description": "Sets up a Node.js environment and optionally installs it to the PATH. Additionally provides proxy support.",
+    "helpUrl": "",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "preview": true,
+    "ecosystem": "node",
+    "minimumAgentVersion": "2.144.0",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "satisfies": [
+        "Node"
+    ],
+    "demands": [],
+    "instanceNameFormat": "Use Node $(versionSpec)",
+    "inputs": [
+        {
+            "name": "version",
+            "type": "string",
+            "label": "Version",
+            "defaultValue": "10.x",
+            "required": false,
+            "helpMarkDown": "Version Spec of the version to use.  Examples: 10.x, 10.15.1, >=10.15.0"
+        },
+        {
+            "name": "checkLatest",
```

Comment:
do we plan to add this option to the "use" syntax? If so, input name should align - unless we can correct it later using an "alias"

Reply:
Agreed, we should look at this going forward, probably a no-op for now though.